src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -30,7 +30,7 @@ limitations under the License.
 
     <command source="client" code="0x00" name="LaunchAppRequest" response="LauncherResponse" optional="false">
       <description>Upon receipt, this SHALL launch the specified app with optional data. The TV Device SHALL launch and bring to foreground the identified application in the command if the application is not already launched and in foreground. The TV Device SHALL update state attribute on the Application Basic cluster of the Endpoint corresponding to the launched application. This command returns a Launch Response.</description>
-      <arg name="data" type="CHAR_STRING"/>
+      <arg name="data"  type="OCTET_STRING"/>
       <arg name="application" type="Application"/>
```

Comment:
These args are in the wrong order (or need explicit field ids, or ideally both).  Please make sure that gets tracked/fixed?